### PR TITLE
Story page - w.i.p

### PR DESCRIPTION
**Pull request by lastDunadan**
Można powoli sprawdzać, ale nowe commity są w drodze.

### CODE REVIEW

File: src/components/BookSummary/BookSummary.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {
   FONT_VARIANTS,
   PURPOSE,
 } from '@/components/Button/Button.types';
-import { SlidePicture } from '@/components/SlidePicture/SlidePicture';
+import { StoryPagePicture } from '@/components/StoryPagePicture/StoryPagePicture';
```
Comment (kubaf13) [inline src/components/BookSummary/BookSummary.tsx:9]: add to index.ts in `StoryPagePicture` folder and then
`import StoryPagePicture from '@/components/StoryPagePicture';`

File: src/components/BookSummary/BookSummary.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import {
...
 import { TagList } from '@/components/TagList/TagList';
```
Comment (kubaf13) [inline src/components/BookSummary/BookSummary.tsx:10]: here the same

File: src/components/StoryPage/StoryPage.styled.ts [hunk 3]
```diff
@@ -0,0 +1,303 @@
+import { css, styled } from 'styled-components';
+
+import { FONT_COLOR } from '@/context/SettingsContext/SettingsContext.types';
+import { FONT_SIZE } from '@/context/SettingsContext/SettingsContext.types';
+import { BACKGROUND_VARIANTS } from '@/context/SettingsContext/SettingsContext.types';
+import type { WildcardData } from '@/fables_library/library.types';
+import { LAYOUT_VARIANTS } from '@/fables_library/library.types';
+
+const FlexCenterAlignment = css`
```
Comment (kubaf13) [inline src/components/StoryPage/StoryPage.styled.ts:9]: lower letters `flexCenterAlignment`

File: src/components/StoryPage/StoryPage.styled.ts [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+        `;
+    }
+  }};
+
+  ${({ wildcard }) => {
+    if (wildcard) {
+      return css`
+        position: absolute;
+        overflow: clip;
+        width: ${wildcard.width}% !important;
```
Comment (kubaf13) [inline src/components/StoryPage/StoryPage.styled.ts:243]: ?

File: src/components/StoryPage/StoryPage.styled.ts [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+
+export const ToolboxContainer = styled.div`
+  width: 50dvw;
+  height: 50dvh;
+  background: antiquewhite;
+`;
+
+const ButtonStyle = css`
+  position: fixed;
+  z-index: 999;
```
Comment (kubaf13) [inline src/components/StoryPage/StoryPage.styled.ts:275]: add too `theme`

File: src/components/BookSummary/BookSummary.tsx (hunk 1 above)
Comment (lastDunadan) [reply src/components/BookSummary/BookSummary.tsx:9]: done

File: src/components/BookSummary/BookSummary.tsx (hunk 2 above)
Comment (lastDunadan) [reply src/components/BookSummary/BookSummary.tsx:10]: done

File: src/components/StoryPage/StoryPage.styled.ts (hunk 3 above)
Comment (lastDunadan) [reply src/components/StoryPage/StoryPage.styled.ts:9]: done

File: src/components/StoryPage/StoryPage.styled.ts (hunk 5 above)
Comment (lastDunadan) [reply src/components/StoryPage/StoryPage.styled.ts]: done

**Review by kubaf13 — APPROVED** [review]

**Pull request merged**